src/pages/api/auth-connect/send-identity-validation-code.ts

```diff
@@ -55,16 +56,24 @@ const handler: Handler = (request, response): Promise<void> => {
       })
         .then(async ({ eventId }) => {
           span.setDisclosedAttribute("is validation code sent", true);
-
-          const temporaryIdentity = {
+          console.log("bqh lol");
```

Comment:
🚒

Reply:
As you can tell by the "q", these were mine..

Reply:
Oops 😳